services/network_instantiations/EventDispatcherWithNetworkBsd.cpp

```diff
@@ -61,6 +61,11 @@ namespace services
         datagrams.push_back(datagram);
     }
 
+    bool EventDispatcherWithNetwork::OpenConnections() const
```

Comment:
Possibly rename to 'HasOpenConnections'. I saw the name 'OpenConnections' first and was expecting this to return the current number of open connections before seeing the return type and implementation.

Reply:
I've renamed to ConnectionsOpen(), so that it isn't a verb, but can be used without confusion as "if (ConnectionsOpen())"